jobbergate-agent/README.md

```diff
@@ -4,17 +4,16 @@
 
 To install the package from Pypi simply run `pip install jobbergate-agent`.
 
-
 ## Setup parameters
 
 1. Setup dependencies
   You can use whenever dependency manager you want to. Just run the command below (and the ones following) on behalf of the manager you prefer.
 
   ```bash
-  make dependencies
+  make install
   ```
 
-2. Setup `.env` parameters
+1. Setup `.env` parameters
```

Comment:
```suggestion
2. Setup `.env` parameters
```

Reply:
Good catch. I expected Markdown to adjust it at render time, but that didn't happen. Fixed in [6e83a61](https://github.com/omnivector-solutions/jobbergate/pull/398/commits/6e83a61f553318c3c1cc543f9d78e9e6c274db23)